### PR TITLE
serverless-cloudfront-deployment

### PR DESCRIPTION
**Pull request by elijah-dev**
1) Serverless-finch and serverless-single-page-app plugins are added and configured. 
2) The app can be built and deployed by running npm script command
3) The app is successfully buld and deployed. It's available with cloudfront url https://d3jb14uz3p28ri.cloudfront.net
4) The index.html is not available publicly from s3 http://aws-practitioner-course-frontend-app.s3-website.eu-central-1.amazonaws.com/

### CODE REVIEW

**Comment by vusatui** [comment]
ATTENTION: 
- Both links are broken in PR description. Check please.

File: package.json [hunk 1]
```diff
@@ -44,7 +44,8 @@
     "react-scripts": "3.4.1",
     "serverless": "^2.29.0",
     "serverless-finch": "^2.6.0",
-    "typescript": "~3.7.2",
+    "serverless-single-page-app-plugin": "file:./serverless-single-page-app-plugin",
```
Comment (vusatui) [inline package.json]: Suggestion: You don't need to add this plugin to dependencies list. Why did you do like this?

Comment (elijah-dev) [reply package.json:47]: It was in the learning materials

Comment (elijah-dev) [reply package.json]: It was in the learning materials 
https://videoportal.epam.com/video/B9Ry76DWWwVng509Y8xq
~3 min 30 sec

Comment (vusatui) [reply package.json]: Okay. But there is more convenient and easiest way. You can put file with plugin inside .serverless_plugins folder and then specify this plugin at plugins section in serverless.yml

Screenshot 2022-07-12 at 18 16 54

**Comment by elijah-dev** [comment]
@vusatui 

I fixed the first link, github couldn't redirect to it, for some reason

Second link is not broken, it suppose to return 403

File: package.json (hunk 1 above)
Comment (elijah-dev) [reply package.json]: ok, good to know

**Comment by vusatui** [comment]
Okay, thanks! Regarding the second link, my bad, I forgot about this criteria.

**Comment by vusatui** [comment]
Could you resolve merge-conflicts, please?

**Pull request merged**